Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -776,46 +776,38 @@ extension BrowserViewController: WKNavigationDelegate {
 
     func webView(webView: WKWebView, decidePolicyForNavigationAction navigationAction: WKNavigationAction, decisionHandler: (WKNavigationActionPolicy) -> Void) {
         let url = navigationAction.request.URL
-        
-        var openExternally = false
-        
+
         if let scheme = url.scheme {
             switch scheme {
             case "about", "http", "https":
-                openExternally = false
-            case "mailto", "tel", "facetime", "sms":
-                openExternally = true
-            default:
-                // Filter out everything we can't open.
-                decisionHandler(WKNavigationActionPolicy.Cancel)
+                decisionHandler(WKNavigationActionPolicy.Allow)
                 return
+            default:
+                break;
             }
         }
-        
-        if openExternally {
-            if UIApplication.sharedApplication().canOpenURL(url) {
-                // Ask the user if it's okay to open the url with UIApplication.
-                let alert = UIAlertController(
-                    title: String(format: NSLocalizedString("Opening %@", comment:"Opening an external URL"), url),
-                    message: NSLocalizedString("This will open in another application", comment: "Opening an external app"),
-                    preferredStyle: UIAlertControllerStyle.Alert
-                )
 
-                alert.addAction(UIAlertAction(title: NSLocalizedString("Cancel", comment:"Alert Cancel Button"), style: UIAlertActionStyle.Cancel, handler: { (action: UIAlertAction!) in
-                    // NOP
-                }))
-                
-                alert.addAction(UIAlertAction(title: NSLocalizedString("OK", comment:"Alert OK Button"), style: UIAlertActionStyle.Default, handler: { (action: UIAlertAction!) in
-                    UIApplication.sharedApplication().openURL(url)
-                    return
-                }))
+        if UIApplication.sharedApplication().canOpenURL(url) {
```

Comment:
I don't like this. I think we should follow what Safari does for at least a few specific url schemes:

* `tel` shows an alert: +1234567890 Cancel / Call
* `sms` opens the SMS app directly (no harm because nothing happens until you confirm there)
* `facetime` shows an alert: myapple@id.com Cancel / FaceTime
* There is also a special case for App Store links (there are a ton of special links) that we should handle in a non-generic way I think?

For other schemes i think it is good to show something generic.